sdk/file_sdk/api_impl/stream_handler.h

```diff
@@ -10,8 +10,8 @@ namespace file {
 class DLL_PUBLIC_FILE StreamHandler : public IStreamHandler
 {
 private:
-  std::vector<Tag> mLabels; // holds the labels to commit; Tag will be raplaced with ILabel when impl
-
+  vector<pair<string, string>> mProperties; // holds the labels to commit; Tag will be raplaced with ILabel when impl
+  vector<Tag> mTags;
   time_t GetCurrentTime();
 
```

Comment:
remove from class

Reply:
this is not my class and Assaf asked to fix his code so i just made this class compile so when Assaf will send his PR, you can tell him to fix all of those comments

---
In reply to: [140700751](https://github.com/AzureAD/rms-sdk-for-cpp/pull/168#discussion_r140700751) [](ancestors = 140700751)